tests/5.0/target/test_target_device.c

```diff
@@ -82,11 +89,13 @@ int test_target_device_device_num() {
  
             target_device_num = omp_get_device_num();
         }
+
+        OMPVV_ERROR_IF(target_device_num == host_device_num, "Target region was executed on host," 
+                   "this region should execute on specified target device number");   
```

Comment:
Maybe _(also note the tailing space at the end of the first string)_:
```suggestion
        OMPVV_ERROR_IF(target_device_num != first_device_num, "Target region was not executed " 
                   "on the specified target device number");   
```